python/ray/util/placement_group.py

```diff
@@ -147,6 +148,12 @@ def placement_group(
             will fate share with its creator and will be deleted once its
             creator is dead, or "detached", which means the placement group
             will live as a global object independent of the creator.
+        max_cpu_fraction_per_node: The maximum fraction of CPU cores this placement
```

Comment:
what's the semantics if two placement groups are used, and each requires 0.5 of the cpus? would ray tune ever create more than one placement groups?

Reply:
0.5 is applied on all cpus. So if one placement group allocates a pg that uses 50% of the node CPU, the second pg won't be scheduled